src/components/ContactCard/ContactGroups.jsx

```diff
@@ -20,18 +24,44 @@ class ContactGroupsClass extends React.Component {
     })
     this.props.updateContact(contact)
   }
+  componentDidUpdate(prevProps, prevState) {
+    if (prevState.createdGroup.length !== this.state.createdGroup.length) {
+      document.getElementsByClassName('react-select__menu-list')[0].scrollTop =
```

Comment:
Do you think we could make this work in the `ContactGroupManager`? As it is, we're creating coupling between these 2 components via a class name, which seems fragile.
When a group is added, the `ContactGroupManager ` receives a props update so in theory it should be possible to do the scroll there?